src/client/pages/OfferNew/Checkout/InsuranceSummary.tsx

```diff
@@ -226,16 +227,26 @@ const getDetails = (
     ])
   }
 
+  const getHouseHoldSizeValue = (
```

Comment:
@palmenhq Is this better?

Reply:
Nice that I made this function really hard to view now by adding comments in the middle of it 🧠

Reply:
Oops I might have deleted your comment by deleting my own comment @palmenhq 😳 What a mess!

Reply:
Haha its ok 😂